pkg/controller/core/capacity_controller.go

```diff
@@ -47,47 +55,126 @@ func NewCapacityReconciler(cache *capacity.Cache) *CapacityReconciler {
 //+kubebuilder:rbac:groups=kueue.x-k8s.io,resources=capacities/finalizers,verbs=update
 
 func (r *CapacityReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	// No-op. All work is done in the event handler.
+	var capObj kueue.Capacity
```

Comment:
Should we do something like: 
```suggestion
	var capObj kueue.Capacity
	capObj..ObjectMeta.Name = req.NamespacedName.Name
```
considering multiple capacities in the cluster, to prevent reconciling on the one we don't want and having to requeue

Reply:
Not sure if I understand your suggestion. `r.client.Get` is already receiving the name of the capacity.

Reply:
ah my bad, I was thinking on something diff, you are right